templates/quickstart-bitbucket-dc-with-vpc.template.yaml

```diff
@@ -70,6 +70,9 @@ Metadata:
           - ESSnapshotId
           - HomeVolumeSnapshotId
           - HomeDeleteOnTermination
+          - BitbucketLicenseKey
```

Comment:
I wonder if we should introduce a new section `Automated setup (Optional)` in the CloudFormation form.